source/Halibut.Tests/Support/LatestClientAndLatestServiceBuilder.cs

```diff
@@ -270,12 +270,14 @@ public async Task<ClientAndService> Build(CancellationToken cancellationToken)
             }
             else if (serviceConnectionType == ServiceConnectionType.PollingOverWebSocket)
             {
+                using var tcpPortConflictLock = await TcpPortHelper.WaitForLock(cancellationToken);
```

Comment:
Funny given our conversation around `using` scopes earlier, but I think an explicit scope with `{ }` would be a bit nicer here, as you could get rid of the lock variable and the scope would be more obvious.

I suppose an explicitly named var _does_ make intent clearer...